src/main/java/fsm/parser/impl/EmployeeLocationExcelParser.java

```diff
@@ -0,0 +1,109 @@
+package fsm.parser.impl;
+
+import fsm.model.domain.Desk;
+import fsm.model.domain.Employee;
+import fsm.model.domain.Floor;
+import fsm.model.domain.Table;
+import fsm.parser.EmployeeLocationFileParser;
+import fsm.parser.exception.AmbigousEmployeeException;
+import fsm.parser.exception.ParserException;
+import fsm.service.EmployeeService;
+import jxl.Cell;
+import jxl.CellType;
+import jxl.Sheet;
+import jxl.Workbook;
+import jxl.read.biff.BiffException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Rohit Singh on 19-12-2016.
+ */
+@Component
+public class EmployeeLocationExcelParser implements EmployeeLocationFileParser {
+
+    @Autowired
+    private EmployeeService employeeService;
+    private Map<String, Employee> deskEmployeeMap;
+
+    public EmployeeLocationExcelParser() {
+        this.deskEmployeeMap = new HashMap<String, Employee>();
+    }
+
+    @Override
+    public void parseEmployeeLocation(File file, Floor floor) {
+        Workbook workbook = this.getWorkbook(file);
+        Sheet sheet = workbook.getSheet(1);
+
+        this.parseDeskEmployeeInfo(sheet);
+        this.mapEmployeeToDesk(floor);
+    }
+
+    private void parseDeskEmployeeInfo(Sheet sheet) {
+        for (int row = 1; row < sheet.getRows(); row++) {
+            Cell deskCodeCell = sheet.getCell(0, row);
```

Comment:
store 0, 1, 2 as private static finals in the parser implementation. Do not use raw constants anywhere